### ISSUE TITLE
Project Name

### ISSUE THREAD

**Issue by selb** [olliez-mods/YummyLife#1]
YumLife was named distinctly from hetuw to avoid implications of continuity or endorsement from the original developers. I would appreciate it if you could similarly name your fork more distinctly from YumLife.

To be clear, despite no longer participating in the Discord communities surrounding OHOL due to personal safety concerns, I do intend to continue work on YumLife, and the two clients will likely be in simultaneous use for some time.

I apologize for not doing more to make YumLife more easily rebrandable before removing Phex support, and appreciate your efforts in this regard.

**Comment by olliez-mods** [comment]
- I will definitely change the icon in one of the upcoming releases.
- As for the names, I haven't seen anyone confuse the two as of yet, but I've tried my best to make them distinct and could add notices/notes on the page and in the program to make it clear you're not involved if it becomes a problem and if you think that would help?

Let me know since I definitely don't want this to become a problem :D

**Comment by selb** [comment]
I first heard about YummyLife from an in-game convo where someone was confused thinking they had YummyLife already but Phex wasn't there (bc they were actually using YumLife), but I think it's probably sinking in already among users and may not be an issue. If you think it will cause more overall user confusion to change YummyLife's name at this point, I wouldn't worry about it.

The only other note of interest is that `yumlife.cfg` is used by both mods, and they will delete each other's non-shared settings if someone switches between. I hit the same issue with hetuw/YumLife and ended up migrating `hetuw.cfg` to `yumlife.cfg` automatically if it was present; it might make sense to migrate to `yummylife.cfg`. That said, I'll probably make a change on the YumLife side soon to preserve unknown config values verbatim and PR it over here.

**Comment by olliez-mods** [comment]
> The only other note of interest is that `yumlife.cfg` is used by both mods, and they will delete each other's non-shared settings if someone switches between.

I hadn't considered that since any removed options would be reset back to defaults each time they switch. 🤔 


If you wanted to do a PR I would definitely pull that change in.
I might just migrate the file name in the end. The reason I haven't already is that I had expected a number of new users (at least in the start) would be people moving from YumLife so I wanted to preserve their settings without them needing to adjust anything... I'll put some thought into it.

I might end up implementing a "backup" data file to restore lost config options on startup if they are found in the backup and not the .cfg

**Comment by selb** [comment]
Yeah, in terms of preserving settings from hetuw the YumLife flow was something like:

- if `yumlife.cfg` is present, just use that
- otherwise, if `hetuw.cfg` is present, load that, but still write out to `yumlife.cfg` for next time
